gitops-v2/build/main.yaml

```diff
@@ -94,6 +102,31 @@ stages:
               condition: and(succeeded(), eq(${{ parameters.dockerLint.enable }}, true))
           - bash: |
               set -ex
+              # Download
+              wget https://github.com/ZupIT/horusec/releases/download/${HORUSEC_TAG}horusec_linux_x86
```

Comment:
Please always `-O` with `wget`:
```
$ wget -O foo https://ifconfig.co 2>&1 | grep Saving
Saving to: ‘foo’
$ wget -O foo https://ifconfig.co 2>&1 | grep Saving
Saving to: ‘foo’
```
Without -O
```
$] wget https://ifconfig.co 2>&1 | grep Saving
Saving to: ‘index.html’
$ wget https://ifconfig.co 2>&1 | grep Saving
Saving to: ‘index.html.1’
```
Note that not only does it create .N files, it also respects the returned resource id. In principle, a malicious actor can overwrite e.g. a go.mod or similar to inject evil code.